erizo_controller/erizoJS/erizoJSController.js

```diff
@@ -141,6 +143,14 @@ exports.ErizoJSController = function () {
         callback('callback', {type: 'initializing'});
     };
 
+    closeWebRtcConnection = function (wrtc) {
+        var associatedMetadata = wrtc.metadata || {};
+        wrtc.close();
+        log.info('message: WebRtcConnection status update, ' +
```

Comment:
In an ideal world that would be the case, and that was what I was aiming for. 
However, it creates race conditions that we want to avoid. In general, you can assume that the connection is no longer useful or active with this log, and you can diagnose the complete destruction of the objects in lower level logs.